tools/clang/lib/SPIRV/Utils.cpp

```diff
@@ -0,0 +1,53 @@
+//===-- Utils.cpp - SPIR-V Utils --------------------------------*- C++ -*-===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "clang/SPIRV/Utils.h"
+#include "llvm/llvm_assert/assert.h"
+
+namespace clang {
+namespace spirv {
+namespace utils {
+
+/// \brief Reinterprets a given string as sequence of words.
+/// Assumes Little Endian architecture.
+std::vector<uint32_t> reinterpretStringAsUintVec(std::string s) {
```

Comment:
FYI: I have implemented this function in #15 since I need it for additional works. It uses similar logic as https://github.com/KhronosGroup/SPIRV-Tools/blob/f18e1f293b7cf8f24a42587b48f35bc49912a5dd/source/text_handler.cpp#L273. Feel free to move the implementation from #15 to this file.